content/series/building-k8s-app-platform/_index.md

```diff
@@ -0,0 +1,96 @@
+      weight: 4
+      pages:
+        - title: Prometheus and Grafana
+          source: "/guides/kubernetes/observability-prometheus-grafana-p1"
+          weight: 1
+        - title: Kubernetes Monitoring Checklist
+          source: "/guides/kubernetes/observability-k8s-monitoring-checklist"
+          weight: 2
+        - title: Kubernetes Platform Monitoring
+          source: "/guides/kubernetes/observability-k8s-platform-monitoring"
+          weight: 3
+        - title: Showback Reference Architecture
+          source: "/guides/kubernetes/observability-showback-refarch"
+          weight: 4
+    - title: Platform Security
+      source: "/guides/kubernetes/platform-security"
+      weight: 5
+      pages:
+        - title: Admission Control Reference Architecture
+          source: "/guides/kubernetes/platform-security-admission-control"
+          weight: 1
+        - title: Secrets and Service Accounts
+          source: "/guides/kubernetes/platform-security-secrets-sa-what-is"
+          weight: 2
+        - title: Secret Management Reference Architecture
+          source: "/guides/kubernetes/platform-security-secret-management"
+          weight: 3
+        - title: Workload Identity
+          source: "/guides/kubernetes/platform-security-workload-identity"
+          weight: 4
+        - title: Developing OPA Policies
+          source: "/guides/kubernetes/platform-security-opa"
+          weight: 5
+    - title: Identity and Access Control
+      source: "/guides/kubernetes/identity"
+      weight: 6
+      pages:
+        - title: Configuring Dex
+          source: "/guides/kubernetes/identity-dex"
+          weight: 1
+    - title: Workload Tenancy
+      source: "/guides/kubernetes/workload-tenancy"
+      weight: 7
+      pages:
+        - title: Platform Readiness Checklist
+          source: "/guides/kubernetes/workload-tenancy-platform-checklist"
+          weight: 1
+        - title: Cluster Acceptance Testing
+          source: "/guides/kubernetes/workload-tenancy-conformance-test"
+          weight: 2
+        - title: Autoscaling Reference Architecture
+          source: "/guides/kubernetes/workload-tenancy-autoscaling-refarch"
+          weight: 3
+        - title: Cluster Tuning Guide
+          source: "/guides/kubernetes/workload-tenancy-cluster-tuning"
+          weight: 4
+        - title: Pod Priority and Preemption
+          source: "/guides/kubernetes/workload-tenancy-priority-preemption"
+          weight: 5
+---
+
+## Building a Kubernetes Application Platform
+
+Kubernetes is a primitive for building platforms at most organizations. Some organizations prefer a turn-key solution such as PAS that can abstract the underlying orchestrator, while others prefer to compose their platform based on cloud native technologies that fit their business needs. The CNFE team's customers typically fall in the latter category. For our customers, this journey includes the following concerns.
```

Comment:
- Replace PAS with Tanzu Application Service
- Remove references to CNFE
- Remove reference to Pacific
- "VMware's Tanzu TKG" stutters
- Remove reference to CNR?